docs/coordinates/apply_space_motion.rst

```diff
@@ -48,3 +48,122 @@ source moves in a straight line with constant velocity in an inertial frame.
+    ...                    pmdec=5.445648092997134,
+    ...                    ref_epoch=2015.0,
+    ...                    phot_g_mean_mag=7.657174523348196)
+
+The 2MASS data for all sources within 1 arcminute around the above position (see
+queries in Note above)::
+
+    >>> result_2mass = dict(RAJ2000=[66.421970000000002, 66.433521999999996,
+    ...                              66.420564999999996, 66.485068999999996,
+    ...                              66.467928999999998, 66.440815000000001,
+    ...                              66.440454000000003],
+    ...                     DEJ2000=[-70.003722999999994, -69.990768000000003,
+    ...                              -69.992255999999998, -69.994881000000007,
+    ...                              -69.994926000000007, -69.993613999999994,
+    ...                              -69.990836999999999],
+    ...                     Jmag=[16.35, 13.663, 16.171, 16.184, 16.292,
+    ...                           6.6420002, 12.275],
+    ...                     Hmag=[15.879, 13.955, 15.154, 15.856, 15.642,
+    ...                           6.3660002, 12.185],
+    ...                     Kmag=[15.581, 14.238, 14.622, 15.398, 15.123,
+    ...                           6.2839999, 12.106],
+    ...                     Date=['1998-10-24', '1998-10-24', '1998-10-24',
+    ...                           '1998-10-24', '1998-10-24', '1998-10-24',
+    ...                           '1998-10-24'])
+
+We'll first create a |SkyCoord| object from the information provided in the TGAS
+catalog. Note that we set the ``obstime`` of the object to the reference epoch
+provided by the TGAS catalog (J2015.0)::
+
+    >>> import astropy.units as u
+    >>> from astropy.coordinates import SkyCoord, Distance
+    >>> from astropy.time import Time
+    >>> c = SkyCoord(ra=result_tgas['ra'] * u.deg,
+    ...              dec=result_tgas['dec'] * u.deg,
+    ...              distance=Distance(parallax=result_tgas['parallax'] * u.mas),
+    ...              pm_ra_cosdec=result_tgas['pmra'] * u.mas/u.yr,
+    ...              pm_dec=result_tgas['pmdec'] * u.mas/u.yr,
+    ...              obstime=Time(result_tgas['ref_epoch'], format='decimalyear'))
+
+We next create a |SkyCoord| object with the sky positions from the 2MASS
+catalog, and an `~astropy.time.Time` object for the date of the 2MASS
+observations provided in the 2MASS catalog (for the data in this region, the
+observation date is the same, so we take just the 0th value)::
+
+    >>> catalog_2mass = SkyCoord(ra=result_2mass['RAJ2000'] * u.deg,
+    ...                          dec=result_2mass['DEJ2000'] * u.deg)
+    >>> epoch_2mass = Time(result_2mass['Date'][0])
+
+We can now use the :meth:`~astropy.coordinates.SkyCoord.apply_space_motion`
+method to compute the position of the TGAS source at another epoch. This uses
+the proper motion and parallax information to evolve the position of the source
+assuming straight-line motion::
```

Comment:
"motion" -> "space motion"?

Reply:
Why?

Reply:
It seems clearer to me that it's a straight line in *real* space rather than "spherical" space... but I see your point.